.asf.yaml

```diff
@@ -21,7 +21,7 @@ github:
   description: >-
     Auxiliary testing data files for Apache GraphAr (Incubating).
   homepage: https://graphar.apache.org/
-  labels:
```

Comment:
this is not the testing file of graphar, it's the configuration of asf repo, so the `labels` is not what we want to change

Reply:
yes, fix it